tests/AttributeBagTest.php

```
@@ -50,72 +51,68 @@ public function testGet(array $attributes, string|UnitEnum $key, mixed $default,
     /**
      * @phpstan-param mixed[] $attributes
      * @phpstan-param mixed[] $values
-     * @phpstan-param mixed[] $expected
      */
     #[DataProviderExternal(AttributeBagProvider::class, 'merge')]
-    public function testMerge(array $attributes, array $values, array $expected): void
+    public function testMerge(array $attributes, array $values, string $expected): void
     {
         AttributeBag::merge($attributes, $values);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\AttributeBag' in method 'testMerge'. (undefined)

(StaticAccess)